federation-js/src/composition/validate/postComposition/executableDirectivesIdentical.ts

```diff
@@ -51,6 +51,7 @@ export const executableDirectivesIdentical: PostCompositionValidator = ({
                 return `\t${serviceName}: ${print(definition)}`;
               })
               .join('\n')}`,
+          directive.astNode || undefined,
```

Comment:
Minor nit: `??` is a more idiomatic way to write this fallback behavior which is fairly recently supported in the version of typescript that we use.
Ref: https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing
```suggestion
          directive.astNode ?? undefined,
```